jsonforms-tooling-common/src/generate.ts

```diff
@@ -23,32 +23,47 @@ export const generateUISchema = async (editorInstance: any, path: string) => {
         filters: {
           'Json Files': ['json'],
         },
+        id: 'selectSchema',
       });
       if (fileUri && fileUri[0].fsPath) {
         path = fileUri[0].fsPath;
       } else {
-        showMessage(editorInstance, 'Please select a json schema file', MessageType.Error);
-        return;
+        throw new Error('Please select a json schema file');
```

Comment:
We shouldn't use exceptions for control flow, rather call `showMessage` and return directly.